modin/pandas/series.py

```diff
@@ -864,7 +864,10 @@ def prod(
     radd = add
 
     def ravel(self, order="C"):
-        return self._default_to_pandas(pandas.Series.ravel, order=order)
+        """
+        Returns the flattened containing data as ndarray.
+        """
+        return self._reduce_dimension(self._query_compiler.ravel(order=order))
```

Comment:
I think we can do this without adding an API to the query compiler, which is ideal to keep that API smaller. This is just `to_numpy` with a flatten. What about this:
```python
return np.flatten(self._query_compiler.to_numpy)
```